setup.json

```diff
@@ -31,10 +31,9 @@
             "sphinxcontrib-details-directive"
         ],
         "pre-commit": [
-            "pre-commit==1.17.0",
-            "yapf==0.28.0",
-            "prospector==1.1.7",
-      	    "pylint==2.3.1"
+            "black==20.8b1",
+            "flake8==3.8.4",
```

Comment:
There's actually no need to install these, as they are installed in special environments in connection with `pre-commit`. I.e., `pre-commit` will take care of installing and running them all on its own using the versions specified in the config file.

Installing them separately here is only relevant if you want to run them separately as well (and not as part of `pre-commit`), which I don't think is necessary.
```suggestion
```

Reply:
Having `pylint` is nice though still as most IDEs use it.

Reply:
Right, I forgot that pre-commit actually installs the dependencies. I'm a bit confused by your suggestion though. Do you want to keep it or not? I actually do run `black` and other linters manually in addition to the hook.

Reply:
The suggested change here is to remove it.
If you run `black` ouside of `pre-commit` often enough that it's useful, then it's fine to keep it - since it will get the same level of "acceptance" as `pylint`.